linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_opensc_installed/rule.yml

```diff
@@ -27,6 +27,7 @@ identifiers:
 
 references:
     disa: "1954"
+    nist: IA-2(4)
```

Comment:
Should this be IA-2(1), IA-2(2), IA-2(3), and IA-2(4)?

ref: https://nvd.nist.gov/800-53/Rev4/control/IA-2